var/spack/repos/builtin/packages/py-metaphlan/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyMetaphlan(PythonPackage):
+    """MetaPhlAn is a computational tool for profiling the composition of
+    microbial communities (Bacteria, Archaea and Eukaryotes) from metagenomic
+    shotgun sequencing data (i.e. not 16S) with species-level."""
+
+    homepage = "https://github.com/biobakery/MetaPhlAn/"
+    pypi = "MetaPhlAn/MetaPhlAn-4.0.2.tar.gz"
+
+    version("4.0.2", sha256="2549fdf2de97a0024551a7bb8d639613b8a7b612054506c88cdb719353f466ff")
+    version("3.1.0", sha256="4e7a7a36d07ed6f4f945afc4216db7f691d44a22b059c2404c917a160a687a6b")
+
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-cython", type=("build", "run"))
```

Comment:
cython doesn't seem to be used by this package

Reply:
It was in the bioconda recipe, so I copied it

Reply:
I don't know what bioconda is, and I definitely don't trust it

Reply:
Bioconda is the most popular conda channel for bioinformatics, and the software authors are the ones that manage the bioconda recipe, so I have no reason to doubt it.

Reply:
So cython is listed as a "host dependency" in the conda recipe, which I am unfamiliar with. Apparently they are "packages that need to be specific to the target platform when the target platform is not necessarily the same as the native build platform"

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

What would the equivalent type be for spack/is it necessary to declare it for spack?

Reply:
When you build the package, is it actually using cython anywhere? Can you upload the build log? There's no mention of cython anywhere in the source code.

Reply:
This log was from a build on a different system, but I don't see it mentioned anywhere

[spack-build-out.txt](https://github.com/spack/spack/files/9693148/spack-build-out.txt)

Reply:
Yep, seems like it's not needed, which makes sense because it isn't used by the build system.